plugins/no-deprecated-colors.js

```diff
@@ -63,6 +63,15 @@ module.exports = stylelint.createPlugin(ruleName, (enabled, options = {}, contex
                 }
               }
             }
+            if (inlineFallback && !replacement.includes(',')) {
+              // report error if inlineFallback is true and replacement does not have a fallback
+              stylelint.utils.report({
+                message: messages.rejected(variableName, replacement, node.prop),
+                node,
+                ruleName,
+                result
+              })
```

Comment:
this is probably fine to copy here, you'll just need a `continue` after so it doesn't run the rest of the code.
```suggestion
              })
              continue
```